components/panel.tsx

```diff
@@ -1,13 +1,22 @@
 interface PanelProps {
   heading: string;
   message: string;
+  email: string | undefined;
```

Comment:
I think we should update message to be {children} and add the text where the declaration is rather than adding an extra dependency here, which makes it single use case and less reusable.

Reply:
That's done now